runtime/base/base.Dockerfile

```diff
@@ -356,7 +356,7 @@ RUN set -xe; \
 #   - OpenSSL
 # Needed by:
 #   - php
-ENV VERSION_POSTGRES=9.6.11
+ENV VERSION_POSTGRES=.10.7
```

Comment:
Is `.10.7` correct? It seems like a weird version number.

Reply:
Yeah. It is correct. It works with the string substitution when creating the url (they changed names of the releases)